.github/workflows/ci.yml

```diff
@@ -0,0 +1,15 @@
+---
+name: CI
+on:
+  push:
+    branches: master
+  pull_request: []
+
+jobs:
+  build-default:
+    name: Build with default Git
```

Comment:
Might want to tweak this name as it's a bit weird for the CI status? Could just be "macOS" or something.